kong/plugins/acme/client.lua

```diff
@@ -524,9 +541,25 @@ local function renew_certificate(premature)
 
     if plugin.name == "acme" then
       kong.log.info("renew storage configured in acme plugin: ", plugin.id)
+
+      -- Create & Save Worker Key
+      -- If worker key already exists, then this means some other worker is renew_config
+      -- Else, continue?
+      local err = store_worker_config(plugin.config)
+      if err and err == "exists" then
+        kong.log.err("Could not place worker lock: " .. ngx.worker.id() .. " with error: " .. err)
+        return
+      end
+
```

Comment:
lets add success logs for key setting and deleting as well with worker id for tracking

Reply:
adding